fetch_system_config/root/lib/udev/rules.d/99-logitechF710.rules

```diff
@@ -1,4 +0,0 @@
-KERNEL=="js?", SUBSYSTEM=="input", ATTRS{idVendor}=="046d", ATTRS{idProduct}=="c219", SYMLINK+="logitechF710"
```

Comment:
No need for these anymore?

Reply:
We could walk back from this, but we've never shipped the Logitech F710 controller or similar with the research robots, and so, one less thing to support here, right?